README.md

```diff
@@ -1034,6 +1034,14 @@ All statements above will return false if used with `===`
 
 <br/><br/><br/>
 
+## ![✔] 7.2. Replicate the Node process
+
+ **TL;DR:** If you really need to use some heavy computational work in your application and you can’t make use of the [non-blocking event loop](https://nodejs.org/en/docs/guides/dont-block-the-event-loop), and if you have a managed server with more than a single CPU, you should consider using the native Node.js `cluster` module or a library like [PM2](http://pm2.keymetrics.io/docs/usage/cluster-mode). Cluster mode will allow you to take advantage of your CPU resources to replicate the single thread when the Event Loop is blocked.
```

Comment:
We want to modernize this guide and refer to modern platform (k8s, serverless). I would start there and mention that most modern platforms have build-in replication capabilities, in case you deploy to a bare server then consider using some... (cluster, PM2)